client/extensions/woocommerce/app/shipping/shipping-methods/free-shipping.js

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
Hmmm. Actually - nevermind - looks like the design is drawing us in this direction. This will be interesting to connect to the REST API